wallet/res/navigation/nav_coinjoin.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright 2023 Dash Core Group.
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/nav_coinjoin"
+    app:startDestination="@id/coinJoinInfoFragment">
+
+    <fragment
+        android:id="@+id/coinJoinInfoFragment"
+        android:name="de.schildbach.wallet.ui.coinjoin.CoinJoinInfoFragment"
+        android:label="CoinJoinInfoFragment"
+        tools:layout="@layout/fragment_coinjoin_info">
+
+<!--        <action-->
+<!--            android:id="@+id/info_to_level"-->
+<!--            app:destination="@id/coinjoinLevelFragment"-->
+<!--            app:enterAnim="@anim/slide_in_right"-->
+<!--            app:exitAnim="@anim/activity_stay"-->
+<!--            app:popExitAnim="@anim/slide_out_left" />-->
```

Comment:
Do we need this action?  If not now, later?

Reply:
Yes, we'll use it later. It's annoying to look for the correct animations so I left them here.